cmd/chart-repo/utils_test.go

```diff
@@ -285,16 +285,29 @@ func Test_parseRepoIndex(t *testing.T) {
 func Test_chartsFromIndex(t *testing.T) {
 	r := repo{Name: "test", URL: "http://testrepo.com"}
 	index, _ := parseRepoIndex([]byte(validRepoIndexYAML))
-	charts := chartsFromIndex(index, r)
+	charts := chartsFromIndex(index, r, new(filters))
 	assert.Equal(t, len(charts), 2, "number of charts")
+	// filter on name
```

Comment:
could we split this up into sub-tests using `t.Run`? (e.g. https://github.com/helm/monocular/blob/4043fe3fd98409933f8500559cec216e49af92a5/cmd/chartsvc/handler_test.go#L73, but without the for-loop)

Reply:
I broke them out into separate tests but did not use a loop using r.Run